test-dssspell-forge.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -e
+
+[[ "$(seth chain --rpc-url="$ETH_RPC_URL")" == "ethlive"  ]] || { echo "Please set a mainnet ETH_RPC_URL"; exit 1;  }
+
+if [[ -z "$1" ]]; then
+  forge test --fork-url "$ETH_RPC_URL" --libraries "src/DssSpell.sol:DssExecLib:0xfD88CeE74f7D78697775aBDAE53f9Da1559728E4" -vvv --force --match "$1"
```

Comment:
we could consider adding `--fork-block-number` as well in the script, as forge supports block pinning.
ref. https://github.com/gakonst/foundry/issues/385#issuecomment-1006945805

Reply:
applied in 776ba76